crates/bevy_pbr/src/atmosphere/functions.wgsl

```diff
@@ -118,6 +118,23 @@ fn sample_transmittance_lut(r: f32, mu: f32) -> vec3<f32> {
     return textureSampleLevel(transmittance_lut, transmittance_lut_sampler, uv, 0.0).rgb;
 }
 
+//should be in bruneton_functions, but wouldn't work in that module bc imports. What to do wrt licensing?
```

Comment:
I think we should just leave a comment saying that the license in the `bruneton_functions.wgsl` file applies to this function, or come up with some way to reorganize the imports. Also just in general, since the code is not exaclty the same as bruneton's verison, it is a WGSL re-write, not sure how the licensing works because I am no legal export. but does it still apply?

Reply:
If it's a direct port with no significant practical changes then I'd say the license still applies.